source/minio-cli/minio-mc/mc-legalhold.rst

```diff
@@ -30,60 +30,207 @@ mode on the object.
    See :mc-cmd-option:`mc mb with-lock` for documentation on creating
    buckets with object locking enabled. 
 
+Quick Reference
```

Comment:
@poornas @vadmeste mc legalhold was out of date (on my local), so I've updated + added versioning. Let me know if this looks right.